Sources/SwifterSwift/SceneKit/SCNVector3Extensions.swift

```diff
@@ -118,6 +126,43 @@ public extension SCNVector3 {
     static func * (scalar: SceneKitFloat, vector: SCNVector3) -> SCNVector3 {
         return SCNVector3(vector.x * scalar, vector.y * scalar, vector.z * scalar)
     }
+    
+    /// SwifterSwift: Divide a SCNVector3 with a scalar
+    ///
+    ///     SCNVector3(10, 20, -30) / 3 -> SCNVector3(3/10, 0.15, -30)
+    ///
+    /// - Parameters:
+    ///   - vector: SCNVector3 to divide.
+    ///   - scalar: scalar value.
+    /// - Returns: result of division of the given SCNVector3 with the scalar.
+    static func / (vector: SCNVector3, scalar: SceneKitFloat) -> SCNVector3 {
+        return SCNVector3(vector.x / scalar, vector.y / scalar, vector.z / scalar)
+    }
+
+    /// SwifterSwift: Divide self with a scalar
+    ///
+    ///     SCNVector3(10, 20, -30) /= 3 -> SCNVector3(3/10, 0.15, -30)
+    ///
+    /// - Parameters:
+    ///   - vector: self.
+    ///   - scalar: scalar value.
+    /// - Returns: result of division of the given CGPoint with the scalar.
+    static func /= (vector: inout SCNVector3, scalar: SceneKitFloat) {
+        // swiftlint:disable:next shorthand_operator
+        vector = vector / scalar
+    }
+
+    /// SwifterSwift: Divide a scalar with a SCNVector3
+    ///
+    ///     3 / SCNVector3(10, 20, -30) -> SCNVector3(3/10, 0.15, -30)
+    ///
+    /// - Parameters:
+    ///   - scalar: scalar value.
+    ///   - vector: SCNVector3 to divide.
+    /// - Returns: result of division of the given CGPoint with the scalar.
+    static func / (scalar: SceneKitFloat, vector: SCNVector3) -> SCNVector3 {
```

Comment:
This function makes no mathematical sense. You can't divide a scalar by a vector. Please remove it. You'll also need to remove the test and the reference in CHANGELOG.